image-patching/stopgap-script/nuage_overcloud_full_patch.py

```diff
@@ -165,30 +201,41 @@ def create_repo_file(reponame, repoUrl, image, gpgcheck):
               'echo "enabled = 1" >> /etc/yum.repos.d/nuage.repo \n'
               'echo "gpgcheck =  %s" >> /etc/yum.repos.d/nuage.repo \n'
               'EOT' % (reponame, repoUrl, gpgcheck)])
-    virt_customize_run('create_repo -a %s --memsize %s --selinux-relabel' % (image, VIRT_CUSTOMIZE_MEMSIZE))
+    virt_customize_run(
+        'create_repo -a %s --memsize %s --selinux-relabel' % (
+            image, VIRT_CUSTOMIZE_MEMSIZE))
 
 
 #####
 # Function to clean up the repo file
 #####
 def delete_repo_file(image):
-    virt_customize('"rm -f /etc/yum.repos.d/nuage.repo" -a %s --selinux-relabel' % (image))
+    virt_customize(
+        '"rm -f /etc/yum.repos.d/nuage.repo" -a %s --selinux-relabel'
+        % (
+            image))
```

Comment:
why image is on seperate line ? it can be changed to `% (image))`

Reply:
done